forecasting/src/autogluon/forecasting/task/forecasting/predictor.py

```diff
@@ -0,0 +1,182 @@
+import copy
+import logging
+
+import pandas as pd
+
+from .dataset import TimeSeriesDataset
+from core.task.base import BasePredictor
+from core.utils import plot_performance_vs_trials, plot_summary_of_models, plot_tabular_models, verbosity2loglevel
+from ...learner import AbstractLearner as Learner  # TODO: Keep track of true type of learner for loading
+from ...trainer import AbstractTrainer  # TODO: Keep track of true type of trainer for loading
+from core.utils.utils import setup_outputdir
+
+__all__ = ['ForecastingPredictor']
+
+logger = logging.getLogger()  # return root logger
+
+
+class ForecastingPredictor(BasePredictor):
+
+    def __init__(self, learner):
+        """ Creates TabularPredictor object.
+            You should not construct a TabularPredictor yourself, it is only intended to be produced during fit().
+
+            Parameters
+            ----------
+            learner : `AbstractLearner` object
+                Object that implements the `AbstractLearner` APIs.
+
+            To access any learner method `func()` from this Predictor, use: `predictor._learner.func()`.
+            To access any trainer method `func()` from this `Predictor`, use: `predictor._trainer.func()`.
+        """
+        self._learner: Learner = learner  # Learner object
+        self._trainer: AbstractTrainer = self._learner.load_trainer()  # Trainer object
+        self.output_directory = self._learner.path
+        self.eval_metric = self._learner.eval_metric
+
+    def get_model_names(self):
+        """Returns the list of model names trained in this `predictor` object."""
+        return self._trainer.get_model_names_all()
+
+    def predict(self, data, model=None):
+        forecasts, tss = self._learner.predict(data, model=model)
+        return forecasts, tss
+
+    def evaluate(self, data, quantiles=None):
```

Comment:
can `evaluate()` instead only take `data` as argument and then also take `kwargs`? Then `quantiles` can simply be one of the possible `kwargs`. This will make it more extensible in the future.